source/docs/appendices/release-notes.md

```diff
@@ -17,6 +17,15 @@ Compatible with:
 - **Magento1**: CE 1.7+, EE 1.12+, [OpenMageLTS](https://www.openmage.org/supported-versions.html) 19.4+
 - **Proximis**: latest version
 
+## 2.11.0
+
+> With its 2.11 release, Front-Commerce continues its exploration of the wide world of B2B commerce! You’ll now be able to deal with Company accounts and company users on Magento 2. A big step for us, knowing the ever-increasing popularity if B2B e-commerce websites.
```

Comment:
Just a remark, we are not consistent between "You'll" and "You will" in the documentation.

Here it is more visible as lines 22 and 24 are not consistent

Reply:
that's right (actually this is the very beginning of the blog post)

and for the record:

```shell
git grep 'will' | wc -l
781
git grep "['’]ll" | wc -l
29
```